modules/vaos/reporting/logs_processor.rb

```diff
@@ -6,7 +6,6 @@
 require 'time'
 require 'json'
 require 'pp'
-require 'digest'
```

Comment:
Do you know what the story is on this?

Reply:
Yeah it was just a duplicate require. Already required on line 4 [here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/vaos/reporting/logs_processor.rb#L4)